toonz/sources/image/sgi/filesgi.cpp

```diff
@@ -1136,7 +1136,7 @@ void SgiWriter::open(FILE *file, const TImageInfo &info) {
       (TBoolProperty *)(m_properties->getProperty("RLE-Compressed"));
   assert(bp);
   bool compressed = bp->getValue();
-  p               = (TEnumProperty *)(m_properties->getProperty("Endianess"));
+  p               = (TEnumProperty *)(m_properties->getProperty("Endianness"));
```

Comment:
The term "Endianess" here is saved in the scene file and used as a key for loading the file format property.
It's fine to fix the typo, but I think we need to prepare some way to accept both the terms "Endianess" and "Endianness" in order to keep backward compatibility.
It will be quite cumbersome work and I think it is out of scope of this PR, so I recommend to keep this typo unfixed.